question/app.js

```diff
@@ -0,0 +1,180 @@
+
+//Q1
+let nickname = 'ゆうし';
+let aged = '23';
+
+let text='私のニックネームは'+nickname+'です。'+'年齢は'+aged+'です。'
+
+console.log(text);
+
+//Q2
+let names =['JavaScript','PHP','Ruby','Python','Go'];
+
+let like = '私の好きな言語は'+names[0]+'です。'+'次は'+names[3]+'を勉強してみたいです。';
+console.log(like);
+
+//Q3
+let user = {
+  name: 'John',
+  age: 26,
+  bloodType: 'A',
+  favorite: 'card',
+};
+console.log(user.age);
+
+//Q4
+let playerList = [
+  {
+    name: 'John',
+    age: 26,
+    favorites: ['Card Game', 'Basket Ball', 'Programming'],
+  },
+  {
+    name: 'Bob',
+    age: 33,
+    favorites: ['Tinder', 'The Legend of Zelda'],
+  },
+  {
+    name: 'Michael',
+    age: 22,
+    favorites: ['Football', 'Smash Bros.'],
+  },
+];
+
+const targetname = "Bob";
+const foundperson = playerList.find(person => person.name === targetname);
+if (foundperson){
+const Bobfavorites = foundperson.favorites;
+console.log(Bobfavorites[1]);
+
+}
+
+
+//Q5
+const totalage = playerList.reduce((sum,player) => sum + player.age, 0);
+const numberofplayers = playerList.length;
+let averageage = 0;
+if (numberofplayers > 0){
+  averageage = totalage / numberofplayers;
+}
+console.log(averageage);
+
+
+//Q6
+const sayWorld = function(){
+  console.log('World');
+}
+sayWorld();
+
+function sayHello(){
+  let greet = 'Hello';
+  console.log(greet);
```

Comment:
基礎編Q6-2

こちら、Q6-1のように1行でも記載可能ですね！
```suggestion
  console.log( 'Hello');
```